hermes/file_gen_test.go

```diff
@@ -25,18 +28,18 @@ type FileNameTestUnit struct {
 	want     string
 }
 
-var test_unit_table [5]FileNameTestUnit
+var testUnitTable [5]FileNameTestUnit
```

Comment:
nit: It's somewhat more normal to define these kind of structs in place (because they often require different things for want & input)

Reply:
Alicja is handling comments on file_gen.go and file_gen_test.go in a different PR, so I'll add this comment there.

Reply:
Ah, it's already there!